### PR TITLE
Render date time

### PR DESCRIPTION
**Pull request by genagain**
This PR tackles #11. The date and time are rendered above the prompt. In addition, I refactored the render method, so it's a lot simpler.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	</head>
 	<body>
 		<nav class="prompt">
-
+      <header class="prompt__header js-header"></header>
```
Comment (genagain) [inline index.html:19]: I'm not sure why the formatting here looks weird on Github, because in vim it looks fine. I'm looking into it now.

Comment (genagain) [reply index.html:19]: I just checked in Atom and it looks fine too. I also noticed the formatting for the submit button looks off too on Github but fine in Atom and Vim. I'm think it's an encoding issue that's outside of our control

Comment (mikehwagz) [reply index.html:19]: Yeah, i notice that github always fucks around with indentation stuff. strange bug.

**Comment by mikehwagz** [comment]
👍

**Review by mikehwagz — COMMENTED** [review]
looks great

**Pull request merged**